inc/save.php

```diff
@@ -38,8 +49,16 @@ function save() {
 		if ( ! empty( $result['errors'] ) ) {
```

Comment:
Above, change the `validate_line()` call to

```php
$result      = validate_line( $line, $line_number, $has_placeholder_record );
```

(Sorry, I can't suggest as it's outside the diff)